ci/lib/install_functions.inc.sh

```diff
@@ -833,7 +866,7 @@ build_example_pkgconfig() {
   pushd "$(mktemp -d)" || return 1
 
   # shellcheck disable=SC2046
-  gcc "${rnpsrc}/src/examples/generate.c" -ogenerate $(pkg-config --cflags --libs $pkgflags librnp) $gccflags
+  ${CC-gcc} "${rnpsrc}/src/examples/generate.c" -ogenerate $(pkg-config --cflags --libs $pkgflags librnp) $gccflags
```

Comment:
ditto

Reply:
CC="" will force cmake default which would be gcc. If CC is unset it will also force cmake default but it may fail at this line depending on shell settings.

Reply:
Here, when `CC` is an empty string, `${CC-gcc}` would actually remain an empty string.  [See my comment above.](https://github.com/rnpgp/rnp/pull/1666#discussion_r745410042)